kinetic/distribution.yaml

```diff
@@ -17531,6 +17541,16 @@ repositories:
       url: https://github.com/clearpathrobotics/wireless.git
       version: master
     status: maintained
+  wit-imu-driver:
```

Comment:
The repo doesn't have a top level license file, could you add one?
Same question about being a fork.

Reply:
OK,I have notified my colleagues to add, and I will reply to you after adding.

Reply:
Now,It has a license

Reply:
thank you